### PR TITLE
Added what BEM stands for (Block Element Modifier)

### PR DESCRIPTION
**Pull request by emmanuel-455**
Majority of developers won't know what BEM stands for. So I just added it.

### CODE REVIEW

File: css-in-javascript/README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@
 
   - Use an underscore for modifiers to other styles.
 
-    > Why? Similar to BEM, this naming convention makes it clear that the styles are intended to modify the element preceded by the underscore. Underscores do not need to be quoted, so they are preferred over other characters, such as dashes.
+    > Why? Similar to BEM (Block Element Modifier), this naming convention makes it clear that the styles are intended to modify the element preceded by the underscore. Underscores do not need to be quoted, so they are preferred over other characters, such as dashes.
```
Comment (ljharb) [inline css-in-javascript/README.md:37]:
```suggestion
    > Why? Similar to [BEM](https://getbem.com/introduction/), this naming convention makes it clear that the styles are intended to modify the element preceded by the underscore. Underscores do not need to be quoted, so they are preferred over other characters, such as dashes.
```

**Review by ljharb — COMMENTED** [review]
Thanks! How about a link instead of just expanding the acronym?

**Review by emmanuel-455 — COMMENTED** [review]
Prefect

File: css-in-javascript/README.md (hunk 1 above)
Comment (emmanuel-455) [reply css-in-javascript/README.md]: > ```suggestion
>     > Why? Similar to [BEM](https://getbem.com/introduction/), this naming convention makes it clear that the styles are intended to modify the element preceded by the underscore. Underscores do not need to be quoted, so they are preferred over other characters, such as dashes.
> ```

Done

Comment (emmanuel-455) [reply css-in-javascript/README.md]: Hello sir I've made the changes

Comment (emmanuel-455) [reply css-in-javascript/README.md]: > ```suggestion
>     > Why? Similar to [BEM](https://getbem.com/introduction/), this naming convention makes it clear that the styles are intended to modify the element preceded by the underscore. Underscores do not need to be quoted, so they are preferred over other characters, such as dashes.
> ```

Hello sir, I've made the changes

**Review by anandfresh — APPROVED** [review]

**Review by ljharb — APPROVED** [review]

**Pull request merged**